### ISSUE TITLE
`release()` checks for 'r_github_packages' in .travis.yml

### ISSUE THREAD

**Issue by wibeasley** [hadley/devtools#912]
Would it be helpful if `release()` checks that the Travis (or AppVeyor or Wercker) builds are *not* using a package version from GitHub?

I haven't run into this trouble (yet) but I could see it happening.  The builds could work perfectly on the local machine and Travis (which have the pre-release version of a package), but not on the CRAN farm (which is uses the old/release version of a package).

The rough steps for the new `release()` step would be:
* Search for .travis.yml (and appveyor.yml & wercker.yml) yaml files. 
* If found, determine if the `r_github_packages` section is populated. 
* If populated, list the packages, and then ask the user if that's intentional (in a no/yup/nope format).

If this feature sounds helpful, I'm happy to start coding it.

**Comment by hadley** [comment]
I don't see that as a big problem, as you should pick up when running `build_win()`

**Comment by wibeasley** [comment]
I agree it's not a big problem, and it's unusual that an error/warning arises in one OS and not the other (which is relevant b/c win-builder won't catch a Linux-specific problem).

I understand if you don't think it happens frequently enough to justify the extra step.  Tell me if your threshold shifts in the future.

------
And I really appreciate the `release()` function.  I updated all four of my CRAN packages in the past week, and was losing track of what changes I made to each package.  This function caught two things that slipped through the cracks.
